client/app/core/event-notifications.service.js

```diff
@@ -1,5 +1,5 @@
 /** @ngInject */
-export function EventNotificationsFactory($timeout, lodash, CollectionsApi, Session, $log, ActionCable) {
+export function EventNotificationsFactory($timeout, lodash, CollectionsApi, Session, $log, ActionCable, ServerInfo) {
   var state = {};
   var toastDelay = 8 * 1000;
```

Comment:
IS this needed since we removed toast?  (not sure if it is used for something else, but should probably be renamed to reflect that if that is the case).